src/orion/benchmark/assessment/averagerank.py

```diff
@@ -38,4 +38,6 @@ def analysis(self, task, experiments):
             algorithm_name = list(exp.configuration["algorithms"].keys())[0]
             algorithm_groups[algorithm_name].append(exp)
 
-        return rankings(algorithm_groups)
+        figures = dict()
+        figures[rankings.__name__] = rankings(algorithm_groups)
```

Comment:
So in `base.analysis` we would get `figures[study.assess_name + "_" + study.task_name][rankings.__name__]`? Do I understand correctly that `study.assess_name` would be  `AverageRank`? Seems there would be duplicate information `[AverageRank_TaskName][rankings]`.

Reply:
there should not be duplicated as same `assessment` and `task` should only combine as a study one time.